src/client/components/HealthCareApp.jsx

```diff
@@ -158,20 +158,23 @@ class HealthCareApp extends React.Component {
     if (submissionStatus === false) {
       submitButton = (
         <ProgressButton
+            additionalClass="submit-status-not-submitted"
```

Comment:
That's exactly the case. If this has no value, we can remove in lieu of `text` checking. I got class crazy last night.